packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -588,6 +588,44 @@ const conf = (module.exports = convict({
     env: 'SOURCE_MAP_TYPE',
     format: String,
   },
+  statsd: {
```

Comment:
We'll need to add notes to the [deploy doc](https://docs.google.com/document/d/1lc5T1ZvQZlhXY6j1l_VMeQT9rs1mN7yYIcHbRPR2IbQ/edit) about these config vars - but I see a placeholder in the doc so :+1: